src/glue.rs

```diff
@@ -1,20 +1,27 @@
-#![cfg(feature = "sled-storage")]
 use {
     crate::{
-        ast::Statement, execute, parse, plan, storages::SledStorage, translate, Payload, Result,
+        ast::Statement,
+        execute, parse, plan,
+        store::{GStore, GStoreMut},
+        translate, Payload, Result,
     },
     futures::executor::block_on,
+    std::{fmt::Debug, marker::PhantomData},
 };
 
-pub struct Glue {
-    pub storage: Option<SledStorage>,
+pub struct Glue<T: Debug, U: GStore<T> + GStoreMut<T>> {
+    _marker: PhantomData<T>,
+    pub storage: Option<U>,
```

Comment:
This was not because of any special reason, it was for `Glue::execute` to use `&mut self`.
`execute` in `executor` module requires ownership of the storage so..

`self.storage` is consumed but `self` is not supposed to be consumed, then.. `Glue` needs something to hold after `storage` is taken.